pyomo/repn/standard_repn.py

```diff
@@ -867,9 +867,9 @@ def _collect_linear(exp, multiplier, idMap, compute_values, verbose, quadratic):
     for c,v in zip(exp.linear_coefs, exp.linear_vars):
         if v.fixed:
             if compute_values:
-                ans.constant += multiplier*v.value
+                ans.constant += multiplier * c * value(v)
```

Comment:
@ruaridhw If `compute_values` is True, then we probably want `value(c)` in case c is a parameter.

Reply:
Good point, thanks!